datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -9885,6 +9885,21 @@ type MLModelGroup implements EntityWithRelationships & Entity & BrowsableEntity
     privileges: EntityPrivileges
 }
 
+"""
+Represents lineage information for ML entities.
+"""
+type MLModelLineageInfo {
```

Comment:
let me know if this is valid approach @shirshanka

Reply:
This looks right, just needs to have the backend mapping on the resolvers.